sync/gh.py

```diff
@@ -79,15 +79,25 @@ def set_status(self, pr_id, status, target_url, description, context):
 
     def add_labels(self, pr_id, *labels):
         logger.debug("Adding labels %s to PR %s" % (", ".join(labels), pr_id))
+        pr_id = self._convert_pr_id(pr_id)
         issue = self.repo.get_issue(pr_id)
         issue.add_to_labels(*labels)
 
     def remove_labels(self, pr_id, *labels):
         logger.debug("Removing labels %s from PR %s" % (labels, pr_id))
+        pr_id = self._convert_pr_id(pr_id)
         issue = self.repo.get_issue(pr_id)
         for label in labels:
             issue.remove_from_labels(label)
```

Comment:
We also need to handle the GithubException where the label already exists (and might be worth special casing the caller to not call this at all if the error is already `None`.

Reply:
Agreed, I've updated